utils/auth/auth.js

```diff
@@ -0,0 +1,22 @@
+import jwt from "jsonwebtoken";
+import dotenv from "dotenv";
+
```

Comment:
I saw some part of the code using `module.exports` and `require` and here you use `import` if node now supports import and export i think you should be consistent throughout the codebase